content/en/docs/how-tos/external-images.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Using External Images in CI"
+date: 2020-10-05T11:14:39-04:00
+draft: false
+---
+
+The `ci-operator` config only allows to reference `ImageStreamTags`, it does not allow to specify arbitrary Docker pull specs. In order
+to use external images, they need to be imported. For this, create a file in the
+[`openshift/release` repository in the `core-services/supplemental-ci-images` folder](https://github.com/openshift/release/tree/master/core-services/supplemental-ci-images)
+that looks like the following:
+
+
+{{< highlight yaml >}}
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: boskos
+  namespace: ci
+spec:
+  tags:
+  - name: latest
+    from:
+      kind: DockerImage
+      name: gcr.io/k8s-staging-boskos/boskos:latest
+    importPolicy:
+      scheduled: true
+{{< / highlight >}}
+
+This will make the image available in the `ci` namespace with as `boskos` with the `latest` tag as soon as your PR merges. Openshift will check every
```

Comment:
Could you add an example of using this in `base_images` for completeness?

Reply:
updated